pkg/target/http.go

```diff
@@ -176,19 +177,21 @@ func newHTTPTarget(
 		messageByteLimit:   messageByteLimit,
 
 		requestTemplate: requestTemplate,
+		approxTmplSize:  approxTmplSize,
 	}, nil
 }
 
-func loadRequestTemplate(templateFile string) (*template.Template, error) {
+func loadRequestTemplate(templateFile string) (int, *template.Template, error) {
```

Comment:
Could we check and error in case the `requestByteLimit` or the `messageByteLimit` set by the user in config are less than the `approxTmplSize`? Probably means a user mistake, but in that cases all messages will count as oversized.

Reply:
Sure, pushed a change. Likely if their limit is that low it'll all be oversized regardless, but no harm in adding the check